test/prettier.js

```diff
@@ -65,6 +65,11 @@ ruleTester.run('prettier', rule, {
       code: `var foo = {bar: 0};\n`,
       filename: getPrettierRcJsFilename('bracket-spacing'),
       options: [{ bracketSpacing: false }, { usePrettierrc: false }]
+    },
```

Comment:
Can you add a test for the parser blacklist behavior? We don't need to actually include processors, but maybe one way to do it would be to put JavaScript code in a file with the `graphql` extension, and assert that no parsing errors are produced (since the text should be parsed as JavaScript).

Reply:
Good call. I tried doing this with including a plugin, but I couldn't get the plugin to be applied within a RuleTester. Forcing the extension is a good middle ground